module-types/Cisco/C9400-SUP-1XL-Y.yaml

```diff
@@ -28,3 +33,6 @@ interfaces:
     type: 25gbase-x-sfp28
   - name: TwentyFiveGigE{module}/0/5
```

Comment:
I see the interface before this one is `TwentyFiveGigE{module}/0/1`, and then it goes to `TwentyFiveGigE{module}/0/5`... Is that correct or should this be `TwentyFiveGigE{module}/0/2`?

Reply:
Now fixed in [f102fb1](https://github.com/netbox-community/devicetype-library/pull/1329/commits/f102fb180ec96a0e0586af4d6266c29062a947b3)